src/Kriti/Parser/Spans.hs

```diff
@@ -27,6 +27,9 @@ alexStartPos = AlexSourcePos 1 1
 data Span = Span {start :: AlexSourcePos, end :: AlexSourcePos}
   deriving (Show, Read, Eq, Ord, Generic)
 
+emptySpan :: Span
+emptySpan = Span (AlexSourcePos 0 0) (AlexSourcePos 0 0)
```

Comment:
If this exists, could we just define a monoid?

Reply:
Unfortunately it doesn't form a valid monoid.